rules/do-you-reliably-deliver-your-tasks/rule.md

```diff
@@ -17,9 +20,8 @@ Some tasks are either time-critical or you give a promise to do them promptly. I
 
 If you're not going to be able to deliver a task on-time, you should let the appropriate people know right away.
 
-
 ::: good  
 ![Figure: Some tasks are time critical. If you have agreed on something then notify the person when you know you will miss the deadline.](../../assets/tickingclock.JPG)  
 :::
 
-You could use your Inbox as a priority list by sending yourself email with an estimate and the priority. Also you are supposed CC other who you think should know about this, so that they could give you some advice and know what is going on here. (Link:        [do-you-send-yourself-emails)](/dones-do-you-send-yourself-emails)
+You could use your Inbox as a priority list by [sending yourself emails](/send-to-myself-emails) with an estimate and the priority. Also you are supposed Cc other who you think should know about this, so that they could give you some advice and know what is going on.
```

Comment:
Change to:
    And Cc...

Reply:
Hi @adamcogan 
you comment is not very clear, but I re-wrote that sentence. Please check again ;)